tests/acceptance/expected-failures-with-ocis-server-owncloud-storage.md

```diff
@@ -51,7 +51,6 @@ Other free text and markdown formatting can be used elsewhere in the document if
 -   [webUIFilesDetails/fileDetails.feature:51](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIFilesDetails/fileDetails.feature#L51)
 
 ### [Sharing seems to work but does not work](https://github.com/owncloud/ocis/issues/1303)
```

Comment:
This file should be renamed `expected-failures-with-ocis-server-ocis-storage.md`
And the same change somewhere in `.drone.star`